internal/faustjs.org/docs/guides/pagination.mdx

```diff
@@ -0,0 +1,99 @@
+---
+slug: /guides/pagination
+title: Pagination
+description: Add pagination to your Faust.js site
+---
+
+This guide covers how to set up Pagination in your Faust.js site with Apollo and WPGraphQL.
+
+## Setup
+
+To get started, create a folder in the root of your Faust.js project called `plugins`.  In this `plugins` folder, create a file called `RelayStylePagination.js` and insert this content:
+
+```
+import { relayStylePagination } from "@apollo/client/utilities";
+
+export class RelayStylePagination {
```

Comment:
```suggestion
export class RelayStylePaginationPlugin {
```

Reply:
Let's name it including "plugin" just to be more clear.

Reply:
Good call